.github/workflows/pipeline.yml

```diff
@@ -221,11 +221,15 @@ jobs:
       NETWORK: aws
     steps:
       - uses: actions/checkout@v3
+      - name: Download actual contracts
+        run:  python3 ./.github/workflows/deploy.py contracts --head_ref_branch=${{ github.head_ref }}
```

Comment:
Maybe better move contract download to neon-tests and run this step: "Download actual contracts" inside container?

Reply:
you are right